website_files/CaseStudies/2025/Supporting the African STACK Community_ The INNODEMS Model.md

```diff
@@ -0,0 +1,159 @@
YFWAKlgvpDD4ciipQF+nkW5KRPT9RSzCLCQTVhXdRNYmJkCNqGrAks1PFyFCa//V4CFzPojYGEmC7V/OAnaO5gAQXgIV8FAUVoAZm1sjQKzKJDNI7BQCCz0XOjgqDHdZfTvAQsv0VWsVAxI2i4D8rARM90zHiGFjZJpAbP2jqYBWPB3kT5ZjA3ETs+2AUO/56cIqp7AzUS2nn8EWHSZO+i319u0/vpmB4QOC378IPQvgKe3D6dBhGj24lI4IlCosm8y6w/6iZrgA6uCk4MguhoLHBa2TsbiCYSjrQqMgaebqPojYAGtSCaO+vb9uUYwWTe9Fb4XIcUUM6vFdFikMTzlsBBq5GZGd1aUbTMhqtwBmEVuY/SSf/dslRb8+QCsR5u9v32z+/Z06dtnK4+359wcWM63Hdy5Pi4sd2rDdwIswAocff7soOcHzPVXN1cYC3N1lsw4EvFBrGciyjaqQUArU3qsKV2TnUaAlZkWh8rK0GSlxwOtMtPiczOTVa7eajee+DbHycH5FtvNwYXj5eohZHtGcjykbK8wlpe7MxdsF4DSwdlVyuWFOrFFLK7QxcOP5eHDcnN1cgkTiVWKEKVcEKcKByKkJESlJUan6qPSkqQZyYpMgyIrLYYoWQYC8wWix3A0pcrBiOVna4rM2pKcuFJzfIVVV12QVJ2XUpQZB84rXSGKCPAIF/BEQbxgH8/CjNQwYTCAPDKULxX5N1QZ05NkkZLApPgoABNQCWTQylCpCYRT5IxODlfxhsQ4KdANvtScFisR+gfwfLhsp8wU0nPZ09MzMiw0XiUPD/Yd6Cgfm6wFh0XANGdDFVPMMgUsXKDBBMyamWmenGygz9C6zixcSDhAA4sQanMIgUVjiwbWzt4EzgZixp1GFbNP1rWONAyo0d2QN7fHgFkALCARoAoBh2O0Y+iwMJpDJKHPYnZ2Z6IKx5jtwl7y+Ak3e8aDmcKwEUQ7LFrwcnN3DHfiwf1ZT8+AWaSDOwgsFRwfPSNbhOHGq78DrJu743z7aAaEP07bNqsQ9VAIePd04cXxKHZ8/7wvDpWzt6e0bZNxP1OY+9u3pKsMbttFfNa1kJC0dUdsIRYRWEw8IWjsuScGeuwTBfjt6BDJjAEVCcIRoIBvwSe/LlKWhUhFMX2WPXdGJbBIXRXZLwfjQYIke8N7Wvjt9I6z9gWGNjG/BWgFkeB3Txa/PZ375tHcz2f7xTzBbkKGPjjM1c2TzfEASHHYHiAEltjVUycJAU6lplMAytCYjAkQ5QGeGIrNSo0FWkH4k6iN1sVJ9fFRRNpIOCbrYhK1Mp0sTO/skXXbx+LES3T18bvNcoRwz4nt6eQa5OQWyvISOXlyHFwBVbcc2V+xXQ2egjxPXgI4LzY3nOMTxPZyZ7l95eDs7eUhjxLLpIFKmSiOWK0wvSYsKV4ClgccEAxAKdoIUHK8BIT9jkHJWimcTNNHGZPl2alKgEguODISSKqLs7VAMashfKA2s8wglwv9pKIAabB3U7o6LiTIg3U7JkLYWJlVUZAsEfrq9JGApLREBe6TCpwCpesVGYlK5BRcQpMFtIKX8GChIi5l65JIV6wgrxytNN8QGxbkU5WfEynwqilJO97tHRqtnJ6jIsHZtqnZZhDTW9EO6zO2FjpR8/MEUiMj1UAuGlUIL9pqUepcJasISWMszGehw8KkO4o5Y7i7M46lWCCaXCCszLKJauyHQp+FQt+ErdyXVvoBc8ga9FbAI3hJhYrMOUQbsG76KRQznIT4kaHP2/MwhZuPIbZwzPRccBICxvWt0ZP7S7bNdchWOjaBpQJI0fvaP4S3PNugZNu7EPQnJgiYgMCfot3CLP/yEozV5Ot7Y8TgQKhIZbVoSKHzooW4wc/5cLEE+ni5DILPAf14Dr/zJQw80bBQxeLkl0zLhkjGTojMPPc3j2ZAbx5Ov6G8HkAWOPLd49nDuUZALe5ChtuRIYLxTvuHL4AQW1izDoKYjorXbPvooOgz34NFerryLdgi6r2ot4/nQfQYSAS3fTjb/OFsA/Tu2Rroh4uN9xckjfX+EuLBtfvrfSQZf7X1/NGCA+vWjFDVxQsLvO3mx/H29vBnO4Ol8Iji+qRxeRCjxbr56JQRmcmxaYZYY5LKlBpnyUjINeryshLhaM3Uw9GUHp+VqklPUsJPFEV7EPoHrFPIE1k+xlvu6c5uRhcPo6u/muMnYHt7szlsBxen22wWy5NMFDqwbzuwICTkOLLCOX4uTiywV55sL3gkcH9OLPdbTiyIpCLDRYroEIi2YmNCdXEAx+j0lFhwc2DrYAD4AIik6WJAwBEQYgX4Ag9jTAG2xpkBu9mJ+TnJRbmpZHecHF2hWZ+hjx5qMnZWpLQWG8BtmTUSQ0xwX0NOY3lSoTlBFhYQHxOamhCdkayCD4EjMgv+OvjbQXAGL+FVEJyEq/BSHRVUatI2FadWmTTl6apqs76lPHus17q30nTvoG+wt7C7t6JvoLp/sAbU018N6uqpAHV0l4J6B6tBPQNVoM6eMhCeAfUNkT1yRkZqu7uLFhfbJibqJyfrxieqQBMTtVSGqx40O9sEmptrBmF6HisbNsmmXr3b2/2UPuewcEXhCQRoVP8GENmNgloajR3+bG3dqZZ+2PQdRFZNA5uofg94J5wESmLjGoz4mGjbJdn9cXRM9np6giT0VvjtqGvnKYShX2Nmx+wBKdHRPSLcggy3I2MKEXbx8gCzWgCy0+ebzGwXXMUe8Oi8Th4u4Q7SiDDQn5jQwRAMcYNAQe/z89Uq0Or9s3kYoIGyQYqRb2KGlsyr6FlwwpG5zReGnH/7ZhfCT2ZSH2XrscUIVJlAZD4zSbFRz3m62Q23MSPEzy6MEU4yc1XM1JV9vpKUOOC0IDZNxooH7C3DdEyIPCAdJrA+kpL69R8vtnGn6B+vdkAfr/ZwovDD1fbF0RiA7KfX+0Cxj4+2eW6+Xl4+Dv/moPYKkPsERngFhHB9hR6+0e5eFm6ApyNL5smLV
```

Comment:
@jumazevick did you intend to commit the base64 encoding of the image within the markdown as well?   This looks like a mistake!